test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -674,3 +674,26 @@ test('wrong type of the blur event (GH-947)', function () {
     eventSimulator.blur(domElement);
     ok(raised);
 });
+
```

Comment:
Add comment that firefox does not support `textInput` properties.
Rename test to `text input` and place it right after the `drag and drop events` test.